### PR TITLE
rsc_2fa 1.2.0 (new formula)

### PR DESCRIPTION
**Pull request by ghost**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/rsc_2fa.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# typed: false
+# frozen_string_literal: true
+
+# Homebrew formula for 2fa
```
Comment (SMillerDev) [inline Formula/rsc_2fa.rb:4]:
```suggestion
```

Comment (log-to-code) [reply Formula/rsc_2fa.rb]: Thank you the linter wanted those lines.

File: Formula/rsc_2fa.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  license "BSD-3-Clause"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args, "-mod=mod", "-o", bin/"2fa"
+  end
+
+  test do
+    out = shell_output("#{bin}/2fa -help 2>&1 || true")
```
Comment (SMillerDev) [inline Formula/rsc_2fa.rb:15]: If this has a non zero return code you can add it as the second argument to shell_output.

```suggestion
    out = shell_output("#{bin}/2fa -help 2>&1")
```

**Review by SMillerDev — APPROVED** [review]
Thanks @log-to-code  ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Pull request closed without merging**